app/assets/javascripts/discourse/app/lib/flag-targets/flag.js

```diff
@@ -0,0 +1,57 @@
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default class Flag {
+  flaggingTopic() {
```

Comment:
you use `isXXXX` for editable and just `xxxxx` for flaggingTopic, maybe we should use is in both cases here